packages/ag-charts-community/src/chart/series/cartesian/scatterSeries.ts

```diff
@@ -680,13 +680,17 @@ export class ScatterSeries extends CartesianSeries<SeriesNodeDataContext<Scatter
         return format;
     }
 
-    getDatumId(datum: ScatterNodeDatum) {
-        const keys = [`${datum.xValue}`, `${datum.yValue}`];
-        if (this.labelKey) keys.push(datum.label.text);
-        return createDatumId(keys);
-    }
+    // getDatumId(datum: ScatterNodeDatum) {
+    //     const keys = [`${datum.xValue}`, `${datum.yValue}`];
+    //     if (this.labelKey) keys.push(datum.label.text);
+    //     return createDatumId(keys);
+    // }
```

Comment:
?

Reply:
Unused code. I commented out because the code that used this method is also commented out in the same file.